app/components/file_upload_preview/file_upload_item/file_upload_item.js

```diff
@@ -145,6 +163,21 @@ export default class FileUploadItem extends PureComponent {
         this.uploadPromise.then(this.handleUploadCompleted).catch(this.handleUploadError);
     };
 
+    downloadFile = async (file) => {
+        const options = {fileCache: true};
+        const tempDownloadFile = await RNFetchBlob.config(options).fetch('GET', file.localPath);
```

Comment:
you should follow the same options as in the image_cache_manager file so we can delete the cache files at our convenience